.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +16,26 @@ Examples:
   * fix(file source): Fix a bug discovering new files
   * chore(external docs): Clarify `batch_size` option
 -->
+
+## Summary
+`Please provide a brief summary about what this PR does.
+This should help the reviewers give feedback faster and with higher quality.`
```

Comment:
```suggestion
<!-- Please provide a brief summary about what this PR does.
This should help the reviewers give feedback faster and with higher quality. -->
```

This might just be a personal preference, but I like comments better than unformatted text for sections users should be filling out.

Reply:
I was looking for an entry box with placeholder text but that's too advanced for markdown. I like the idea of making it a comment so the users don't need to delete it each time 👍